daemon/cluster/cluster.go

```diff
@@ -1570,8 +1529,7 @@ func (c *Cluster) CreateNetwork(s apitypes.NetworkCreateRequest) (string, error)
 	return r.Network.ID, nil
 }
 
-// RemoveNetwork removes a cluster network.
-func (c *Cluster) RemoveNetwork(input string) error {
+func (c *Cluster) updateClusterObject(updateFn func(ctx context.Context, state nodeState) error) error {
```

Comment:
I am not sure if `updateClusterObject` does not contain the following codes
```
        state := c.currentNodeState()
 	if !state.IsActiveManager() {
 		return c.errNoManager(state)		
 	}
```
will the handler return an error of `this node is not part of swarm`.

Then a docker with no swarm mode will response to cluster update action which may break api.